test/test-test262.js

```diff
@@ -97,19 +97,6 @@ function hasExcludedInfo(testInfo) {
       // some tests fail because they rely on a side effect which is failing
       // resolving an issue might uncover another issue
 
-      // 'Statement cannot contain an `import` declaration', // https://github.com/Agoric/transform-module/issues/5
```

Comment:
Let's keep these commented items, at least until we are out of the experimental phase. They are useful when debugging (you can uncomment a few at a time).

Reply:
Reverted.